technical-resources/loans/open-term-loans.md

```diff
@@ -0,0 +1,160 @@
+
+### `callDueDate`
+
+$$
+\
+\text{callDueDate} =
+\begin{cases}
+\text{dateCalled + noticePeriod} & \text{if } \text{dateCalled} > 0 \\
+0 & \text{if } \text{dateCalled} = 0
+\end{cases}
+\
+$$
+
+### `impairedDueDate`
+
+$$
+\
+\text{impairedDueDate} =
+\begin{cases}
+\text{dateImpaired} & \text{if } \text{dateImpaired} > 0 \\
+0 & \text{if } \text{dateImpaired} = 0
+\end{cases}
+\
+$$
+
+### `normalDueDate`
+$$paidOrFundedDate = \max(datePaid, dateFunded)$$
+
+$$
+\
+\text{normalDueDate} =
+\begin{cases}
+\text{paidOrFundedDate + paymentInterval} & \text{if } \text{paidOrFundedDate} > 0 \\
+0 & \text{if } \text{paidOrFundedDate} = 0
+\end{cases}
+\
+$$
+
+## `paymentDueDate()`
+
+$$paymentDueDate = \min(callDueDate, impairedDueDate, normalDueDate)$$
+
+## `_defaultDates()`
+Returns the due dates for `callDefaultDate`, `impairedDefaultDate` and `normalDefaultDate`
+
+### `callDefaultDate`
+
+$$callDefaultDate = callDueDate$$
+
+### `impairedDefaultDate`
+
+$$
+\
+\text{impairedDefaultDate} =
+\begin{cases}
+\text{impairedDueDate + gracePeriod} & \text{if } \text{impairedDueDate} > 0 \\
+0 & \text{if } \text{impairedDueDate} = 0
+\end{cases}
+\
+$$
+
+### `normalDefaultDate`
+$$
+\
+\text{normalDefaultDate} =
+\begin{cases}
+\text{normalDueDate + gracePeriod} & \text{if } \text{normalDueDate} > 0 \\
+0 & \text{if } \text{normalDueDate} = 0
+\end{cases}
+\
+$$
+
+## `defaultDate()`
+
+$$defaultDate = \min(callDefaultDate, impairedDefaultDate, normalDefaultDate)$$
+
+# Loan Calls
+
+The Pool Delegate (PD) exclusively possesses the ability to commence a Loan Call, specifying the principal amount that must not exceed the current principal balance. In the event of partial Loan Calls, the loan continues with a reduced balance but the same terms. Conversely, a complete Loan Call leads to the loan maturing once the borrower repays. Upon a Loan Call by the PD, the borrower is obligated to settle the demanded amount within the Notice Period or risk the loan defaulting.
+
+The PD also has the power to withdraw a Loan Call, which reverts the payment schedule back to its original state prior to the Loan Call. If a Loan Call is actioned on a late loan, the PD has the discretion to default the loan at the termination of either the Notice Period or the Grace Period, depending on which comes first.
+
+It's worth noting that refinancing is considered a valid resolution to Loan Calls, with such calls being settled once the parties agree to new terms.
+
+# Fees
+Open-Term Loans have `delegateServiceFees` and `platformServiceFees` which are prorated and calculated on each payment.
+ 
+# Closing Loan
+
+If a Borrower wants to close a Loan prematurely, they can do so by calling `closeLoan`. This will allow them to pay back their entire principal balance in one transaction. There is one fee parameter used to calculate the fees associated with this action:
+- `closingRate`: Fee charged as a percentage of the outstanding principal at the time of payment.
```

Comment:
There is no `closingRate` or `closeLoan` function in OTLs. Closing the loan is simply done by `makePayment` with a `principalToReturn` matching the outstand principal.